Terminal.Gui/Core/View.cs

```diff
@@ -167,6 +149,27 @@ internal enum Direction {
 		/// </summary>
 		public Action<MouseEventArgs> MouseClick;
 
+		/// <summary>
+		/// Gets or sets the HotKey defined for this view. A user pressing HotKey on the keyboard while this view has focus will cause the Clicked event to fire.
+		/// </summary>
+		public Key HotKey { get => viewText.HotKey; set => viewText.HotKey = value; }
```

Comment:
Is there a way for a View to not have a hotkey specified?  I am thinking about entry lines, or lists, not sure how this is handled in this new setup.

Reply:
Yes, by default the hotkey capability is 'disabled' because the `HotKeySpecifier` defaults to 0xffff.

Reply:
Thanks!